utils/src/com/goide/GoEnvironmentUtil.java

```diff
@@ -47,4 +49,19 @@ public static String retrieveGoPathFromEnvironment() {
     String path = EnvironmentUtil.getValue(GoConstants.GO_PATH);
     return path != null ? path : PathMacros.getInstance().getValue(GoConstants.GO_PATH);
   }
+  
+  @NotNull
+  public static String[] getEnvironment(@NotNull Project project, @NotNull String sdkHomePath) {
+    String[] env = EnvironmentUtil.getEnvironment();
+    
+    for (int i=0; i < env.length; i++) {
+      if (env[i].toLowerCase().startsWith("path=")) {
```

Comment:
It looks like problem of go. So let's invoke ```gofmt``` explicitly with full path instead of ```go fmt```. This magic with path variable looks unreliable.

BTW, it's strange to get map, flattern it and then look for string that starts by key of initial map (see method implementation for details). Also startsWith is case sensitive and can't handle differences between OS. So for such cases use EnvironmentUtil#getValue